source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+
+
+.. _evaluation:
+
+Evaluation
+----------
+
+An evaluation operator evaluates the value of any field in a document.
+
+The evaluation operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Evaluation Method
+     - Matches
+      
+   * - :java-core-api:`mod() <com/mongodb/client/model/Filters.html#mod(java.lang.String,long,long)>`
+     - documents where a modulo operation on the value of a field contain a specified result.
+
+   * - :java-core-api:`regex() <com/mongodb/client/model/Filters.html#regex(java.lang.String,java.lang.String)>`
+     - documents where values contain a specified regular expression.
+      
+   * - :java-core-api:`text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+     - documents which contain a specified full-text search expression.
+     
+   * - :java-core-api:`where() <com/mongodb/client/model/Filters.html#where(java.lang.String)>`
+     - documents which contain a specified JavaScript expression.
+
+The following example matches documents that have a ``color`` field
+starting with the letter "p" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin regexComparison
+   :end-before: end regexComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$regex" : "^p" }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+
+.. _bitwise:
+
+Bitwise
+-------
+
+THe bitwise operator converts a number into it's binary value to
+evaluate its bits.
+
+The bitwise operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Bitwise Method
+     - Matches
+      
+   * - :java-core-api:`bitsAllSet() <com/mongodb/client/model/Filters.html#bitsAllSet(java.lang.String,long)>`
+     - documents when the specified bits of a field are set (i.e. "1").
+
+   * - :java-core-api:`bitsAllClear() <com/mongodb/client/model/Filters.html#bitsAllClear(java.lang.String,long)>`
+     - documents when the specified bits of a field are clear (i.e. "0").
+      
+   * - :java-core-api:`bitsAnySet() <com/mongodb/client/model/Filters.html#bitsAnySet(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are set (i.e. "1").
+     
+   * - :java-core-api:`bitsAnyClear() <com/mongodb/client/model/Filters.html#bitsAnyClear(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are clear (i.e. "0").
+
+The following example matches documents that have a ``bitField ``field
```

Comment:
The formatting breaks here because of the lack of a space after the closing "``"